tutor/resources/styles/components/top-nav-bar.less

```diff
@@ -1,12 +1,43 @@
-@tutor-navbar-height: 61px;
+@tutor-navbar-height: 60px;
 @tutor-navbar-padding-horizontal: 4rem;
 
 // the @navbar-XX vars are bootstrap defaults
 
 .tutor-top-navbar {
+
+  background: @tutor-white;
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.1);
+  color: @openstax-gray;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  height: @tutor-navbar-height;
+  padding-right: @navbar-padding-horizontal;
```

Comment:
I think we it'd be better to use `min-height` instead of `height`.  Using `height` won't allow it to grow just in case we end up with too many large items on it and it needs to expand.